tests/e2e/e2e_util_test.go

```diff
@@ -450,3 +450,357 @@ func (s *IntegrationTestSuite) getLatestBlockHeight(c *chain, valIdx int) int {
+		fmt.Sprintf("--%s=%s", flags.FlagChainID, c.id),
+		"--keyring-backend=test",
+		"--output=json",
+		"-y",
+	}
+
+	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, endpoint)
+	s.T().Logf("%s successfully created group: %s", adminAddr, groupMembersPath)
+}
+
+func (s *IntegrationTestSuite) execUpdateGroupMembers(c *chain, valIdx int, endpoint string, adminAddr string, groupId string, groupMembersPath string, fees string) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	s.T().Logf("Executing gaiad tx group update-group-members %s", c.id)
+
+	gaiaCommand := []string{
+		"gaiad",
+		"tx",
+		"group",
+		"update-group-members",
+		adminAddr,
+		groupId,
+		groupMembersPath,
+		fmt.Sprintf("--%s=%s", flags.FlagGasPrices, fees),
+		fmt.Sprintf("--%s=%s", flags.FlagChainID, c.id),
+		"--keyring-backend=test",
+		"--output=json",
+		"-y",
+	}
+
+	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, endpoint)
+	s.T().Logf("%s successfully updated group members: %s", adminAddr, groupMembersPath)
+}
+
+func (s *IntegrationTestSuite) executeCreateGroupPolicy(c *chain, valIdx int, endpoint string, adminAddr string, groupId string, metadata string, policyFile string, fees string) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	s.T().Logf("Executing gaiad tx group create-group-policy %s", c.id)
+
+	gaiaCommand := []string{
+		"gaiad",
+		"tx",
+		"group",
+		"create-group-policy",
+		adminAddr,
+		groupId,
+		metadata,
+		policyFile,
+		fmt.Sprintf("--%s=%s", flags.FlagGasPrices, fees),
+		fmt.Sprintf("--%s=%s", flags.FlagChainID, c.id),
+		"--keyring-backend=test",
+		"--output=json",
+		"-y",
+	}
+
+	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, endpoint)
+	s.T().Logf("%s successfully created group policy: %s", adminAddr, policyFile)
+}
+
+func (s *IntegrationTestSuite) executeSubmitGroupProposal(c *chain, valIdx int, endpoint string, fromAddress string, proposalPath string) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	s.T().Logf("Executing gaiad tx group submit-proposal %s", c.id)
+
+	gaiaCommand := []string{
+		"gaiad",
+		"tx",
+		"group",
+		"submit-proposal",
+		proposalPath,
+		fmt.Sprintf("--%s=%s", flags.FlagGasPrices, fees),
+		fmt.Sprintf("--%s=%s", flags.FlagFrom, fromAddress),
+		fmt.Sprintf("--%s=%s", flags.FlagChainID, c.id),
+		"--keyring-backend=test",
+		"--output=json",
+		"-y",
+	}
+
+	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, endpoint)
+	s.T().Logf("%s successfully submited group proposal: %s", fromAddress, proposalPath)
+}
+
+func (s *IntegrationTestSuite) executeVoteGroupProposal(c *chain, valIdx int, endpoint string, proposalId string, voterAddress string, voteOption string, metadata string) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	s.T().Logf("Executing gaiad tx group vote %s", c.id)
+
+	gaiaCommand := []string{
```

Comment:
is there no `--from` flag for these commands?

Reply:
no `gaiad tx group vote [proposal-id] [voter] [vote-option] [metadata] [flags]` it's assumed the voter is the `from` address